### DISCUSSION TITLE
Way to update?

### DISCUSSION THREAD

**Discussion by Travillion** [mkb79/Audible#101] [Q&A] [answered]
Hello,

First off, thanks for all your work on this! I've been using audible and audible-cli for several months. This morning I encountered a few errors, the most important of which looks to be:
Error in job: Bad Request (400): 1 validation error detected: Value 'Extreme' at 'quality' failed to satisfy constraint: Member must satisfy enum value set: [High, Normal]
This makes me think the parameters on the back end have changed and I probably need to update audible and audible-cli packages. I am not very experienced with linux. Do I just reinstall the packages, or is there a way to update them?

**Comment by mkb79** [comment, marked as answer]
Hi,

these „bug“ is audible-cli releated. This package is not affected by this.

I've solved this issue in the latest master branch on github. You can clone audible-cli and install it. I need some time before I release the changes to PyPI or make new binarys.

**Comment by Travillion** [comment]
Thank you!
